imports/ui/components/T9n.js

```diff
@@ -15,6 +15,34 @@ Meteor.startup(function() {
     // STUDENT ROLE
     aluno: "Aluno",
     "ex-aluno": "Ex-aluno",
+    // STATES
+    ac: "AC",        
```

Comment:
No, because it should be shorten to fit it better, look:
![image](https://cloud.githubusercontent.com/assets/4422230/19727626/ee55be56-9b6e-11e6-9e40-368153c88aef.png)